src/engine/execute.js

```diff
@@ -148,6 +148,8 @@ const execute = function (sequencer, thread) {
     // Recursively evaluate input blocks.
     for (const inputName in inputs) {
         if (!inputs.hasOwnProperty(inputName)) continue;
+        // Do not evaluate the internal custom command block within definition
+        if (inputName === 'custom_block') continue;
```

Comment:
What do you think about adding an opcode check here? As-is, this would suppress an input named `custom_block` on an extension block, for example. Not that I expect that to be a common case -- it may never happen -- but this check does feel a bit imprecise to me.